opendbc/dbc/tesla_model3_party.dbc

```diff
@@ -223,10 +223,34 @@ BO_ 646 DI_state: 8 ETH
  SG_ DI_locStatusCounter : 8|4@1+ (1,0) [0|0] ""  X
  SG_ DI_locStatusChecksum : 0|8@1+ (1,0) [0|0] ""  X
 
-BO_ 659 DAS_settings: 8 XXX
- SG_ DAS_autopilotEnabled : 38|1@0+ (1,0) [0|1] "" XXX
- SG_ DAS_settingCounter : 52|4@1+ (1,0) [0|15] "" XXX
- SG_ DAS_settingChecksum : 63|8@0+ (1,0) [0|255] "" XXX
+BO_ 659 UI_chassisControl: 8 CH
+ SG_ UI_chassisControlChecksum : 56|8@1+ (1,0) [0|255] ""  epas3s
+ SG_ UI_chassisControlCounter : 52|4@1+ (1,0) [0|15] ""  epas3s
+ SG_ UI_selfParkTune : 48|4@1+ (1,0) [0|15] ""  das
+ SG_ UI_redLightStopSignEnable : 46|2@1+ (1,0) [0|3] ""  das
+ SG_ UI_pedalSafetyEnable : 44|2@1+ (1,0) [0|3] ""  das
+ SG_ UI_ldwEnable : 42|2@1+ (1,0) [0|3] ""  das
+ SG_ UI_latControlEnable : 40|2@1+ (1,0) [0|3] ""  das
```

Comment:
might have no effect, but we should test sending this message on bus 0 and see if we can get it disabled automatically